### PR TITLE
Added badges

### PR DESCRIPTION
**Pull request by psychoactiv**
### Badges

1. Badges can be used to denote any notification.
2. Badges on avatar is used.
3. Badges on Icons and text are use
12 02 2022_15 53 29_REC

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
looks good to me , note giving rem always is not mandatory

File: components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+}
+.avatar-badge-container::before {
+  position: absolute;
+  content: "";
+  bottom: -0.1rem;
+  right: -0.1rem;
+  border-radius: 999rem;
+  padding: 0.5rem 0.5rem;
+  border: 5px solid var(--tertiary);
+
```
Comment (mariyasada) [inline components/Badges/badges.css:181]: remove unnecessary space

**Review by mariyasada — APPROVED** [review]
good to merge

File: components/Badges/badges.css (hunk 1 above)
Comment (psychoactiv) [reply components/Badges/badges.css:181]: Sure

**Pull request merged**